geomstats/_backend/pytorch/__init__.py

```diff
@@ -154,6 +154,10 @@ def flip(x, axis):
     return torch.flip(x, dims=axis)
 
 
+def broadcast_arrays(*args, **kwargs):
+    return torch.broadcast_tensors(*args, **kwargs)
```

Comment:
Couldn't this be imported as broadcast_arrays directly?

Reply:
Yes! thank you for that